sql/create_tables.sql

```diff
@@ -0,0 +1,43 @@
+/* USERS */
+CREATE TABLE users(
+    id BIGSERIAL PRIMARY KEY,
+    display_name VARCHAR(50),
+    first_name VARCHAR(50),
+    last_name VARCHAR(50),
+    elo_score INTEGER,
+    creation_time TIMESTAMP

```

Comment:
I think we should also have an update_time field, which would get updated any time there is a change made. We should do it for all of the tables.